src/Arcus.WebApi.Logging/ExceptionHandlingMiddleware.cs

```diff
@@ -63,7 +63,11 @@ public async Task Invoke(HttpContext context, ILoggerFactory loggerFactory)
             {
                 await _next(context);
             }
-            catch (BadHttpRequestException ex)
+#if NET6_0
+            catch (Microsoft.AspNetCore.Http.BadHttpRequestException ex) 
+#else
+            catch (BadHttpRequestException ex) 
```

Comment:
Should we explicitly put the FQN here as well ?

Reply:
Yeah, we could. Will be a bit cleaner. Thx!

Reply:
Done that! 👍